src/utils/releaseNotes.js

```diff
@@ -30,4 +36,44 @@ function getPrNumbersFromReleaseNotes(releaseNotes) {
   return [...new Set(allPrNumbers)]
 }
 
+exports.getPRBody = (
+  template,
+  { newVersion, draftRelease, inputs, author, artifact }
+) => {
+  const tagsToBeUpdated = []
+  const { major, minor } = semver.parse(newVersion)
+
+  if (major !== 0) tagsToBeUpdated.push(`v${major}`)
+  if (minor !== 0) tagsToBeUpdated.push(`v${major}.${minor}`)
+
+  // Should strictly contain only non-sensitive data
+  const releaseMeta = {
+    id: draftRelease.id,
+    version: newVersion,
+    npmTag: inputs['npm-tag'],
+    opticUrl: inputs['optic-url'],
+  }
+
+  const prBody = template({
+    releaseMeta,
+    draftRelease,
+    tagsToUpdate: tagsToBeUpdated.join(', '),
+    npmPublish: !!inputs['npm-token'],
+    artifact,
+    syncTags: /true/i.test(inputs['sync-semver-tags']),
+    author,
+  })
+
+  if (prBody.length > MAX_PR_BODY_SIZE_LIMITATION) {
+    const omissionText =
+      '. *Note: Part of the release notes have been omitted from this message, as the content exceeds the size limit*'
+    return _truncate(prBody, {
+      length: PR_BODY_TRUNCATE_SIZE,
+      omission: omissionText,
+    })
+  }
```

Comment:
this logic contains a big bug

Reply:
I have moved the metadata to the top of the template so it doesn't get truncated

Reply:
can we please truncate the release notes so that the truncation logic doesn't depend on where the release metadata is positioned in the body of the PR?